tsconfig.json

```diff
@@ -3,7 +3,8 @@
 		"declaration": true,
 		"declarationMap": true,
 		"esModuleInterop": true,
-		"moduleResolution": "node",
+		"module": "ESNext",
+		"moduleResolution": "bundler",
```

Comment:
Note that the current version of vscode doesn't support `"moduleResolution": "bundler"`. The insiders version does though so the stable version should soon.